lm75bd/lm75bd.c

```diff
@@ -25,15 +25,64 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
   return ERR_CODE_SUCCESS;
```

Comment:
looks like theres something wrong in the integration test since its not ending

Reply:
Poked around a little bit on this and not sure what's going on here. Any reason you pointed out this line in particular? Is this something on my end or something that's broken in the CI/CD?

Reply:
Likely something on your end since this has happened before with other PRs where there were errors

Reply:
Nothing in particular about this line lol mb